src/assets/src/models.ts

```diff
@@ -22,6 +22,14 @@ export interface BluejeansMetadata {
     numeric_meeting_id: string;  // Number for dial-in / URL
 }
 
+export interface ZoomMetadata {
+    user_email: string;
+    user_id: number;
+    meeting_id: number;
+    meeting_url: string;
+    numeric_meeting_id: string;  // Number for dial-in / URL
+}
+
```

Comment:
I see just below this that `backend_metadata?` only uses `BluejeansMetadata`. I guess this works because Zoom uses objects with the same properties. Might be worth combining into one interface or specifying an `or`.

Reply:
I was on the fence about that, since they are the same right now, but they don't really need to be. I settled on condensing ZoomMetadata into `interface ZoomMetadata extends BluejeansMetadata {}` to make it clear it can have additional props if necessary, and I corrected `backend_metadata` in `Meeting` to `BluejeansMetadata|ZoomMetadata`.

Reply:
Seems okay. I suppose we'll likely remove `BluejeansMetadata` in the near future.